db/migrations/20240215104631_add_last_introspection_status.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE repositories
+    RENAME COLUMN status to last_introspection_status;
```

Comment:
we typically try to write the up migrations to be idempotent.  You can't do something quite as easy as "IF EXISTS", but there are a couple solutions : https://stackoverflow.com/questions/48102295/rename-column-only-if-exists